build/media_source/com_templates/vue/components/elements/item.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+	<div :class="['item', element.type]">
+		<div class="btn-wrapper">
+			<button type="button" class="btn btn-lg" @click="editElement(element)">
+				<span class="icon-options"></span>
+				<span class="sr-only">{{ translate('COM_TEMPLATES_EDIT') }}</span>
+			</button>
+			<button type="button" class="btn btn-lg" @click="$emit('delete')">
+				<span class="icon-cancel"></span>
+				<span class="sr-only">{{ translate('COM_TEMPLATES_DELETE_ELEMENT') }}</span>
+			</button>
+		</div>
+
+		<div class="item-content">
+			<div class="desc">
+				<span>{{ element.type }}</span>
+				<span v-if="element.options.class">.{{ element.options.class }}</span>
+			</div>
+
+			<grid v-if="element.type === 'Grid'" :grid="element"></grid>
+
+			<item v-else v-for="child in element.children" :item="child" @delete="deleteElement(child)"></item>
```

Comment:
Provide  key props in for loop

Reply:
So, when an update comes it can identify.

Reply:
@Anu1601CS  I don't think we have some unique identifier which can be set as a key prop, as our JSON only has a type, options and children. So can we implement without a key or should we add some key/counter externally?